pom.xml

```diff
@@ -64,21 +64,27 @@
         <cxf.version>3.1.4</cxf.version>
         <cxf.xpp3.bundle.version>1.1.4c_6</cxf.xpp3.bundle.version>
         <jts.bundle.version>1.12_1</jts.bundle.version>
-        <karaf.version>2.4.3</karaf.version>
+        <karaf.version>4.0.4</karaf.version>
         <lux.version>1.2</lux.version>
         <opengis.bundle.version>${org.geotools.version}_1</opengis.bundle.version>
         <org.geotools.bundle.version>${org.geotools.version}_1</org.geotools.bundle.version>
         <org.geotools.version>8.4</org.geotools.version>
         <org.slf4j.version>1.7.1</org.slf4j.version>
```

Comment:
Looks like karaf 4.0.4 is bringing in a newer version of slf4j. It would be nice if we could just pull a lot of these properties out of the karaf parent pom instead of duplicating them in here. Not sure if there's a way to do that other than setting it as a parent.

A better way would probably be to look at how the properties are being used. If the properties are just being used for maven dependencies, you could look at leveraging dependency management by importing them from the karaf parent. So in that case the property itself might not be needed.

`<org.slf4j.version>1.7.1</org.slf4j.version>`